src/requests/features.jl

```diff
@@ -61,21 +60,24 @@ function process(r::JSONRPC.Request{Val{Symbol("textDocument/signatureHelp")},Te
             end
         end
     end
-    (isempty(sigs) || (typof(x) === CSTParser.PUNCTUATION  && kindof(x) === CSTParser.Tokens.RPAREN)) && return send(JSONRPC.Response(r.id, CancelParams(Dict("id" => r.id))), server)
+    if (isempty(sigs) || (typof(x) === CSTParser.PUNCTUATION  && kindof(x) === CSTParser.Tokens.RPAREN))
+        # TODO Is this an error? Or should we return an empty result here?
+        error("Unclear what should happen here.")
+        # return send(JSONRPC.Response(r.id, CancelParams(Dict("id" => r.id))), server)
```

Comment:
Same thing here, it seems to me that sending a `CancelParams` here is not spec conform. Is this an error state? Or should we just send an empty response here?

Reply:
Empty response

Reply:
I'm not entirely clear how to resolve that, can you open a PR that fixes this once we have merged this PR here?

Reply:
Will do